test/anvil.test.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env jest */
+
+const { expect } = require('@jest/globals')
+
+describe('1.8.9 anvil', () => {
+  const Item = require('prismarine-item')('1.8.8')
+  test('combine two damaged sword', () => {
+    const sword1 = Item.fromNotch({ blockId: 276, itemCount: 1, itemDamage: 5, nbtData: { type: 'compound', name: '', value: { ench: { type: 'list', value: { type: 'compound', value: [{ lvl: { type: 'short', value: 1 }, id: { type: 'short', value: 34 } }] } }, RepairCost: { type: 'int', value: 1 } } } })
+    const sword2 = Item.fromNotch({ blockId: 276, itemCount: 1, itemDamage: 3 })
+    const res = Item.anvil(sword1, sword2, false, false)
+    const inverse = Item.anvil(sword2, sword1, false, false)
+    expect(res).toStrictEqual({ finalEnchs: [{ lvl: 1, name: 'unbreaking' }], fixedDurability: 190.32, usedMats: 1, xpCost: 3 })
```

Comment:
Durability is an integer, it cannot be `190.32` if that's the result of your function, you must be missing some rounding or truncation somewhere.

Reply:
done